pkgs/development/tools/minizinc/ide.nix

```diff
@@ -1,11 +1,9 @@
 { lib, mkDerivation, fetchFromGitHub, qtbase, qtwebengine, qtwebkit, qmake, minizinc }:
+
 mkDerivation rec {
```

Comment:
How about switching to `finalAttrs` pattern here?

Reply:
could be done, maybe